crates/hir_def/src/nameres/collector.rs

```diff
@@ -1402,8 +1402,18 @@ impl ModCollector<'_, '_> {
 
         // Case 1: builtin macros
         if attrs.by_key("rustc_builtin_macro").exists() {
+            // `#[rustc_builtin_macro = "builtin_name"]` overrides the `macro_rules!` name.
+            let name;
```

Comment:
Forgot to delete ?

Reply:
No, it is assigned a `tt::Ident` below at line 1410. The next `let name =` then gets a reference to the `tt::Ident` as value at line 1411.

Reply:
TIL ! Neat !